.github/workflows/Release.yaml

```diff
@@ -142,50 +142,14 @@ jobs:
               repo: context.repo.repo,
               ref: `heads/${context.payload.pull_request.head.ref}`,
             })
-  push_docker:
-    needs: release
-    name: Push Docker Image
-    runs-on: ubuntu-22.04
-    timeout-minutes: 720
-    strategy:
-      matrix:
-        rosdistro: [humble]
-        arch: [amd64]
-        # Build test for arm64 CPU is broken.
-        # This is a temporary solution and will be repaired in the future.
-        # See also https://github.com/tier4/scenario_simulator_v2/pull/1295
-        # arch: [amd64, arm64]
-    steps:
-      - name: Free Disk Space (Ubuntu)
-        uses: jlumbroso/free-disk-space@main
-        with:
-          tool-cache: false
-
-      - name: Install docker for ubuntu runner
-        uses: docker/setup-buildx-action@v3
-
-      - name: Install QEMU
-        uses: docker/setup-qemu-action@v3
-
-      - uses: actions/checkout@v3
 
-      - name: Login to GitHub Container Registry
-        uses: docker/login-action@v2
-        with:
-          registry: ghcr.io
-          username: ${{ github.actor }}
-          password: ${{ secrets.GITHUB_TOKEN }}
-
-      - name: Build and push (${{ matrix.arch }})
-        uses: docker/bake-action@v3
-        with:
-          files: |
-            ./docker-bake.hcl
-          workdir: .
-          set: |
-            *.cache-to=type=gha,mode=max
-            *.cache-from=type=gha
-            *.tags=ghcr.io/tier4/scenario_simulator_v2:humble-${{ needs.release.outputs.new_version }}
-          push: ${{ github.event.pull_request.merged == true }}
-          targets: |
-            ${{ matrix.rosdistro }}_base_${{ matrix.arch }}
+      - name: Kick docker build action
+        if: github.event.pull_request.merged == true
+        run: |
```

Comment:
Does this step really work?
What this step is calling is a trigger called repository_dispatch, and it looks like there is no trigger in Docker.yaml.

And, are there any restrictions that I have to call it via curl?
It seems simpler to call Docker.yaml with `uses` like below.

```yaml
      - name: Kick docker build action
        if: github.event.pull_request.merged == true
        uses: ./.github/workflows/Docker.yaml
```

Reply:
I understand that this step does not triggers `repository_dispatch` but `workflow_dispatch`.